src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -116,7 +116,7 @@ trait NodeStatusBuilder {
 trait BlockChainBuilder {
   self: StorageBuilder =>
 
-  lazy val blockchain: Blockchain = BlockchainImpl(storagesInstance.storages)
+  lazy val blockchain: BlockchainImpl = BlockchainImpl(storagesInstance.storages)
```

Comment:
:+1:
Created [EC-288](https://iohk.myjetbrains.com/youtrack/issue/EC-288) for looking into it.